west.yml

```diff
@@ -59,7 +59,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 8182ebc2d257c309b9c50fc0d895a4b88a672a1b
+      revision: pull/1168/head
```

Comment:
`1fbee9f184e0047bc98fe96629dcafe45de5ab09`

Reply:
Thank you